modules/ROOT/pages/dataweave-cookbook-change-value-of-a-field.adoc

```diff
@@ -98,27 +93,21 @@ users: { (payload.users mapObject (userObject) -> {
 </users>
 ----
 
+The `update` operator provides a general way to change the value of a field. However, note that DataWeave provides a simpler way to mask values, without specifying the path to each sensitive field:
```

Comment:
This sentence makes me think that `update` and `mask` are different functions to get the same result, but `update` provides more features to change values than `mask`. If possible, I'd separate these two examples each under their subsection: 
== Example `update`
== Example `mask`

Reply:
I'd say `mask` is a specification of `update`: the updated value is always the same (`****`) and it applies to all matching paths (`password` key).  
The problem here was the example, I kept it the same for simplicity but we should find a better example for `update` that does not involve passwords and the like, as that is exactly the use case for `mask`. 
IMO we should have 2 separate entries (not sections) on the cookbook:
- "Masking sensitive information" with this example and the `mask` solution
- "Changing the value of a field" with a new example and the `update` operator

-